pallets/asset-manager/src/migrations.rs

```diff
@@ -433,3 +433,113 @@ where
 		Ok(())
 	}
 }
+
+pub struct PopulateSupportedFeePaymentAssets<T>(PhantomData<T>);
+impl<T: Config> OnRuntimeUpgrade for PopulateSupportedFeePaymentAssets<T> {
+	fn on_runtime_upgrade() -> Weight {
+		log::info!(target: "PopulateSupportedFeePaymentAssets", "actually running it");
```

Comment:
These log statements are potentially helpful (I think this is copy-pasted from other migrations) but we could probably rephrase them to have more helpful context. Maybe not `info`, too...?